cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/discover_advanced_settings.spec.js

```diff
@@ -221,7 +235,13 @@ describe('discover_advanced_setting', () => {
   });
 
   describe('Number of terms advanced setting', () => {
+    beforeEach(() => {
```

Comment:
Nit: not a big concern, shall we move `beforeEach` to outermost describe? It seems beforeEach be added to each inside describe.

Reply:
Oh, I thought beforeEach only take effect inside a describe, if that's the case then I will sure to move to outermost describe.

Reply:
Thanks for calling it out, I learned a lot from this comment.